javacutil/src/org/checkerframework/javacutil/InternalUtils.java

```diff
@@ -315,8 +315,8 @@ public static TypeMirror leastUpperBound(
      */
     public static TypeMirror greatestLowerBound(
```

Comment:
I think it would be good to mention in the javadoc for these two methods that the unannotated type is lub/glb-ed.
The javadoc currently also doesn't say what the difference to Types.lub/glb is; in case you know why we even have these two methods, please also add that.
Otherwise this looks good and ready to merge.